packages/cli/generators/project/templates/package.json

```diff
@@ -35,7 +35,7 @@
 <% } -%>
     "pretest": "npm run clean && npm run build",
 <% if (project.mocha) { -%>
-    "test": "lb-dist mocha DIST/test",
+    "test": "lb-mocha 'DIST/test'",
```

Comment:
AFAIK, single quotes don't work on Windows, that's why we have to use escaped double quotes in package scripts.

```json
{
  "test": "lb-mocha \"DIST/test\""
}
```